tasks/go.py

```diff
@@ -479,7 +479,7 @@ def reset(ctx):
 
 
 @task
-def generate(ctx):
+def generate(ctx, build_tags=""):
```

Comment:
#5725 changed the way build tags are managed, here and in other places you are reverting to the old behavior. You should merge master into this branch so that this doesn't happen